keps/prod-readiness/sig-network/5495.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 5495
+deprecated:
+  approver: "@soltysh"
```

Comment:
I just put Maciej here as a placeholder. I'm not too sure how to find a PRR reviewer.

Reply:
I DMed him and he's fine with it